packages/datadog-instrumentations/src/google-cloud-pubsub.js

```diff
@@ -0,0 +1,122 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const requestStartCh = channel('apm:google-cloud-pubsub:request:start')
+const requestInjectMsgCh = channel(`apm:google-cloud-pubsub:request:injectMsg`)
```

Comment:
Is this needed? It looks like all messages are available when `start` is called.